.github/workflows/add-pr-to-release-project.yml

```
@@ -0,0 +1,32 @@
+---
+name: "add merged PRs to release party project"
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - closed
+
+jobs:
+  add_pr_to_release_project:
+    if: github.base_ref == 'main' && github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92
```

Why a specific commit instead of a release version that dependabot can track and update?  That commit seems to map to the current release. :)
```suggestion
        uses: tibdex/github-app-token@v1.8.0
```

---

I'm for us pinning versions going forward whenever images are involved and setting the example for the rest of the org.
It's a hella PITA and costs us some automation, but bumping versions of hashed things ourselves isn't too problematic to automate down the line.